bot/api/fetchQueuesByOwner.ts

```diff
@@ -0,0 +1,22 @@
+import { Connection } from "typeorm";
+import { QueueEntity } from "../entities/queue";
+
+export default async (conn: Connection, ownerId: string, channelId: string) => {
+  try {
+    const result = await conn
+      .getRepository(QueueEntity)
+      .createQueryBuilder()
+      .where(`"ownerId" = :ownerId AND "channelId" = :channelId`, {
+        ownerId,
+        channelId,
+      })
+      .getMany();
+    return result;
```

Comment:
Everything looks great. But do you think this API call is best for getting all queues? Or should you also pass in a specific Status that you want - E.g. passing in ACTIVE to get all active queues or SCHEDULED to get all scheduled queues?